### ISSUE TITLE
Nutil creates thumbnails only for some images in a folder

### ISSUE THREAD

**Issue by ingrre** [Neural-Systems-at-UIO/nutil#119]
Not sure what's up with these particular images, but they are not behaving as they should. I have tried making thumbnails of TIFFs, but only  70/150 pngs are created. Tried with both versions of nutil, the new version doing worse than the old (did not try the new beta version though). See files and thumbnails here: `Y:\HBP-Voucher_Rat-corticome\SubProject3_Insula\13018\2-TIFF\TIFF-tiled`. The TIFFs are tiled versions of the images in `Y:\HBP-Voucher_Rat-corticome\SubProject3_Insula\13018\2-TIFF\TIFF-original`.

There is obviously something with the files, and most likely not with Nutil, but I would be very happy if you could help me spot the error.

**Comment by sharoncy** [comment]
Ingrid has tested v033 and v0401, she will try out v0403.

**Comment by ingrre** [comment]
Tried nutil v0403. 
Don't have an error message to display, as Nutil just shut down without telling why (or is this hidden somewhere?). The result was 54 pngs made out of 153 TIFs. Pngs and the .nut-file used are here: `Y:\HBP-Voucher_Rat-corticome\SubProject3_Insula\13018\2-TIFF\TIFF-tiled\thumbnails-nutil4.03`

**Comment by sharoncy** [comment]
@ingrre We suspect that the problem is to do with the network: transforming images located on the network rather than working locally. Nicolaas doesn't have access to the files from home so will have to go to the office to investigate further.

**Comment by ingrre** [comment]
Nutil (all versions) are downloaded to and runs from this location: C:\Users\ingrreit

**Comment by sharoncy** [comment]
@leuat. I did a test overnight, Nutil located locally (via Qt), data on the server (lagringshotell). It generated thumbnails for 49/155 tled tiffs and then crashed: 
![Nutil_debug](https://user-images.githubusercontent.com/30626642/81920502-9183d780-95d9-11ea-8ab7-531a738d3795.PNG)

**Comment by sharoncy** [comment]
I copied 103 of the images to the C drive and ran transform locally. Thumbnails were generated successfully for all 103 images.

**Comment by sharoncy** [comment]
I ran another test, images located on server, nutil in Qt and saved transformed thumbnails to folder located locally on c drive. Nutil crashed after transforming 44/155 images.

**Comment by sharoncy** [comment]
I did another test. Images on server, nutil in Qt, and saved the transformed thumbails to a folder on the server. I ran Nutil with one core only. It crashed after transforming 119/155 images.

**Comment by sharoncy** [comment]
Let's contact IT and see if they can do anything about this.

**Comment by leuat** [comment]
This one is now merged with issue #132

**Comment by sharoncy** [comment]
I suspect there is a corrupted file in this dataset and that it is not a server problem. I ran a transform test with images located on the "lagringshotell" that are known not to be corrupted and it successfully transformed all images (>100 images, took 6 + hours). I then repeated this with this dataset (also on lagringshotell) it crashed (after approx 7 min, tested twice). How do we identify the corrupted file? this needs investigation @leuat
